gradle.properties

```diff
@@ -8,4 +8,4 @@ name = logisim-evolution
 #  x.y.z-suffix (i.e. "3.6.9-beta1")
 # Suffix is optional and can be any string indicating **non-stable** releases
 # (i.e. "beta1" or "rc1"). Keep suffix EMPTY string for every stable release!
-version = 3.5.0
+version = 3.7.0
```

Comment:
Shouldn't this be `3.6.0`?

Reply:
I already branched of with 3.6.0 this morning (branch rc_3_6_0). the 3.7.0 is for the "new" development branch